src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs

```diff
@@ -255,9 +256,6 @@ public class CryptoConfig
                 ht.Add("1.3.6.1.4.1.311.88.2.1", "System.Security.Cryptography.Pkcs.Pkcs9DocumentName, " + AssemblyName_Pkcs);
                 ht.Add("1.3.6.1.4.1.311.88.2.2", "System.Security.Cryptography.Pkcs.Pkcs9DocumentDescription, " + AssemblyName_Pkcs);
 
-                s_defaultNameHT = ht;
-                return s_defaultNameHT;
```

Comment:
I'm having unrelated problems building corefx at present (in System.ComponentModel.csproj among others - I'm trying to get past that, but may have to dig out my Windows machine where these problems rarely surface 😄)